bin/ocr-transform.sh

```diff
@@ -50,6 +49,8 @@ main () {
         fi
     fi
 
+    declare -a script_args
```

Comment:
Why did you move that line?

Reply:
I simply wanted to have it close to the first use of `script_args`.